### PR TITLE
Log grpc messages only after encoding them

### LINKED ISSUE #2634: Ensure all gRPC message are logged exclusively using `TextFormat` encoding
### Problem

To reduce our risk from yet-to-be-found vulnerabilities in log4j2, we should only log gRPC messages in octal-encoded format.

That is, before logging any gRPC message, we should:
1. Internally convert it to a `ByteString`.
2. Log it via `TextFormat.escapeBytes()`.

### Solution

1. Add a helper function that accepts a `logger and log level` or a `consumer`, `message` string and `gRPC` message, and performs the logging with an octal-encoded message.
2. Ensure all gRPC messages logged in _hedera-node/_ go through this helper.

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by anighanta**
Signed-off-by: anighanta <anirudh.ghanta@hedera.com>

**Description**:

**Related issue(s)**:

Fixes #2634

**Notes for reviewer**:
Added a Log utility that will encode different GRPC types that we log before actually logging them.

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-node/src/main/java/com/hedera/services/context/BasicTransactionContext.java [hunk 1]
```diff
@@ -174,7 +174,7 @@ public AccountID submittingNodeAccount() {
 		try {
 			return nodeInfo.accountOf(submittingMember);
 		} catch (IllegalArgumentException e) {
-			log.warn("No available Hedera account for member {}!", submittingMember, e);
+			log.warn("No available Hedera account for member {}! : {}", submittingMember, e.getMessage());
```
Comment (tinker-michaelj) [inline hedera-node/src/main/java/com/hedera/services/context/BasicTransactionContext.java:177]: Imo we should leave the stack trace in a `WARN`-level message like this one.

**Review by tinker-michaelj — COMMENTED** [review]
This looks like a reasonable approach to me! There are multiple failing [unit tests](https://app.circleci.com/pipelines/github/hashgraph/hedera-services/28191/workflows/9dfc634b-855a-4077-8249-5bda1ef3e069/jobs/97927) on this branch atm.

**Comment by anighanta** [comment]
> This looks like a reasonable approach to me! There are multiple failing [unit tests](https://app.circleci.com/pipelines/github/hashgraph/hedera-services/28191/workflows/9dfc634b-855a-4077-8249-5bda1ef3e069/jobs/97927) on this branch atm.

fixed the unit test already. Working on code smells now.

File: hedera-node/src/main/java/com/hedera/services/ledger/accounts/AliasManager.java [hunk 2]
```diff
@@ -71,7 +71,7 @@ public void rebuildAliasesMap(final MerkleMap<EntityNum, MerkleAccount> accounts
 				aliases.put(v.getAlias(), k);
 			}
 		});
-		log.info("Rebuild complete : No.of accounts with aliases {} ", aliases.size());
+		log.debug("Rebuild complete : No.of accounts with aliases {} ", aliases.size());
```
Comment (Neeharika-Sompalli) [inline hedera-node/src/main/java/com/hedera/services/ledger/accounts/AliasManager.java:74]: I think it helps to be at `info` level and helps while loading state.

File: hedera-node/src/main/java/com/hedera/services/utils/LogUtils.java [hunk 3]
```diff
@@ -0,0 +1,305 @@
+package com.hedera.services.utils;
+
```
Comment (Neeharika-Sompalli) [inline hedera-node/src/main/java/com/hedera/services/utils/LogUtils.java:2]: license header missing

File: hedera-node/src/test/java/com/hedera/services/utils/LogUtilsTest.java [hunk 4]
```diff
@@ -0,0 +1,300 @@
+package com.hedera.services.utils;
+
```
Comment (Neeharika-Sompalli) [inline hedera-node/src/test/java/com/hedera/services/utils/LogUtilsTest.java:2]: same here

**Comment by tinker-michaelj** [comment]
Out-of-date.

**Pull request closed without merging**